cypress/objects/LoginPage.js

```diff
@@ -0,0 +1,66 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+
+    getLoginFormButton() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+
+    getTimesheets() {
+        return cy.get('.page__title');
+    }
+
+    getCalendar() {
+        return cy.get('.calendar');
+    }
+    
+    getTileForm() {
+        return cy.get('.tile.form');
+    }
+
+    getLogin() {
+        return cy.get('[id="loginForm.role"]');
+    }
+    
+    getMainNamLink() {
+        return cy.get('.main-nav__link--active');
+    }
+    getUserInfo() {
+        return cy.get('.user-info__title');
+    }
+
+    getMainNav() {
+        return cy.get('.main-nav');
+    }
+
+    getUrl() {
+        return cy.url();
```

Comment:
this function is kind of redundant, it does not take parameter but just calls another function, would be better to call that function directly, what I would suggest:

validateUrl() {
      cy.url().should('include', '/time-logging')
    }

but then this should be in another page object